apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -1029,6 +1064,49 @@ export class InstallManager {
     });
   }
 
+  private _mergeEnvironmentVariables(
+    baseEnv: NodeJS.ProcessEnv,
+    environmentVariables?: IConfigurationEnvironment
+  ): NodeJS.ProcessEnv {
+    const packageManagerEnv: NodeJS.ProcessEnv = baseEnv;
+
+    if (environmentVariables) {
+      // eslint-disable-next-line guard-for-in
+      for (const envVar in environmentVariables) {
+        let setEnvironmentVariable: boolean = true;
+        console.log(`\nProcessing definition for environment variable: ${envVar}`);
+
+        if (process.env[envVar]) {
+          setEnvironmentVariable = false;
+          console.log(colors.yellow(`WARNING: Environment variable already defined:`));
+          console.log(`  Name: ${envVar}`);
+          console.log(`  Value set on the device: ${process.env[envVar]}`);
```

Comment:
```suggestion
          console.log(`  Value set on the device: ${baseEnv[envVar]}`);
```

Reply:
@iclanton What is a better way of phrasing "Value set on the device"? I mean, even if the variable is not defined on the device, rush can still be launched with with this variable defined.

Reply:
"Existing value"?

"device" doesn't really make sense in this context.